vcd/datasource_vcd_nsxt_network_context_profile.go

```diff
@@ -0,0 +1,55 @@
+package vcd
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+func datasourceVcdNsxtNetworkContextProfile() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdNsxtNetworkContextProfileRead,
+
+		Schema: map[string]*schema.Schema{
+			"context_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Context ID can be one of VDC Group ID, ",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
```

Comment:
same comment - not needed ForceNew in all fields.

Reply:
Removed.